src/arrayMethodJoin.js

```diff
@@ -5,7 +5,21 @@
  */
 function applyCustomJoin() {
   [].__proto__.join2 = function(separator) {
-    // write code here
+    let resultingString = '';
+    const splitter = separator === undefined ? ',' : separator;
```

Comment:
you can set default value for parameter 'separator' like that: function(separator = ',') {} if function accepts nothing